pcsx2/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ add_library(PCSX2_FLAGS INTERFACE)
 if (NOT PCSX2_CORE)
 	add_executable(PCSX2)
 else()
-	add_library(PCSX2)
+	add_library(PCSX2 OBJECT)
```

Comment:
Should probably gate this with the simd check, that way local builds don't take a speed hit.

Reply:
Sure